doc/train-fitting-tensor.md

```diff
@@ -0,0 +1,125 @@
+# Train a Deep Potential model to fit `tensor` like `Dipole` and `Polarizability`
+
+Unlike `energy` which is a scalar, one may want to fit some high dimensional physical quantity, like `dipole` (vector) and `polarizability` (matrix, shorted as `polar`). Deep Potential has provided different API to allow this. In this example we will show you how to train a model to fit them for a water system. A complete training input script of the examples can be found in 
+
+```bash
+$deepmd_source_dir/examples/water_tensor/dipole/dipole_input.json
+$deepmd_source_dir/examples/water_tensor/polar/polar_input.json
+```
+
+The training and validation data are also provided our examples. But note that **the data provided along with the examples are of limited amount, and should not be used to train a productive model.**
+
+
+
+The directory of this examples:
+
+-   [The training input script](# the-training-input-script)
+-   [Training Data Preparation](# training-data-preparation)
+- 	[Train the Model](# train-the-model)

```

Comment:
The hyperlink is not working
plz refer to https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#links for help

Please make sure the links are working before PR, thanks!